From 38323f13ae317a218fa5cc18632aee171615814e Mon Sep 17 00:00:00 2001
From: Alfred Chen <cchalpha@gmail.com>
Date: Thu, 16 Jan 2020 11:03:50 +0800
Subject: [PATCH 057/106] bmq: Update recommend SCHED_TIMESLICE

Recommend 2 in ms for PREEMPT system and 4ms for others.
---
 init/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/init/Kconfig b/init/Kconfig
index 5afd5a419665..6f47609e6b61 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -704,7 +704,8 @@ config SCHED_TIMESLICE
 	depends on SCHED_BMQ
 	help
 	  Time slice in ms for BMQ CPU scheduler, default 4 ms.
-	default 4
+	default 2 if PREEMPT
+	default 4 if !PREEMPT
 
 config UCLAMP_TASK
 	bool "Enable utilization clamping for RT/FAIR tasks"
-- 
2.27.0.rc2

